README.md

```diff
@@ -41,7 +41,8 @@ If you are new here, please read the getting started docs:
 
 * [logs](./docs/logs/getting-started/README.md)
 * [metrics](./docs/metrics/getting-started/README.md)
-* [traces](./docs/trace/getting-started/README.md)
+* traces: [ASP.NET Core](./docs/trace/getting-started-aspnetcore/README.md) |
```

Comment:
Hmm, I suggest keep this unchanged, by default users will go to the simple console example (assuming a new user just came and wanted to understand the basics, and the user doesn't have to mix this with the actual integration of OpenTelemetry and an application framework). In the console tutorial doc, I think we can add something in the beginning "this is showing the basics of a simple console application, if you want to learn how to use OpenTelemetry with ASP.NET Core, read X, if you want to learn how to use UWP, read Y, ..."

Reply:
Can we do the reverse? Link to the ASP.NET Core case with a comment for simple console/.NET Framework? I'm assuming more people will be interested in the ASP.NET Core style.

Reply:
I don't know, here is one thing we need to consider: for folks who want to learn how to extend the SDK or how to configure advanced scenarios, do we want them to get into ASP.NET Core or stay with a basic console app? Or we can ask the question in another way - would C# "Hello, World" choose console or ASP.NET Core?

Reply:
I guess I'm fine with the "getting started with tracing" doc pointing to ASP.NET Core demo, and in the banner put something like "hey if you don't want to bother with HTTP hosting, here goes a simple console version you want to use" (it seems a bit awkward to me, but I don't see big no).

Reply:
> for folks who want to learn how to extend the SDK or how to configure advanced scenarios, do we want them to get into ASP.NET Core or stay with a basic console app?

I think this is a good question, and maybe it's true that if a custom SDK extension is your goal then you might want a guide that used a basic console app. Though, I'd say that if you're writing a custom SDK extension, then you're probably already past "getting started" with OTel 😆.

I think our "getting started" users are most likely folks who just want to see some telemetry get emitted from an app, and personally I think the fact entangling our getting started with an application framework actually serves to better meet this need - i.e., as a new user I don't need to entangle myself with learning anything about the activity api out of the gate.

Reply:
> I think this is a good question, and maybe it's true that if a custom SDK extension is your goal then you might want a guide that used a basic console app. Though, I'd say that if you're writing a custom SDK extension, then you're probably already past "getting started" with OTel 😆.

+1 👍

Reply:
@cijothomas What are your thoughts here? Before I make any changes I want to make sure we're all in agreement.

Reply:
Fwiw here is what I did in the .NET docs: https://learn.microsoft.com/en-us/dotnet/core/diagnostics/distributed-tracing-collection-walkthroughs#collect-traces-using-opentelemetry

I did ASP.NET first, console 2nd. My assumption is that ASP.NET is the more common case users will care about and it also has the advantage that you don't need to add any manual instrumentation to see a result for distributed tracing.

Reply:
I chatted with @cijothomas a bit offline. I think ideally we would only have the ASP.NET Core guide. Agree with @noahfalk it seems like what most users will be after and is nice and concise. Manual instrumentation would be a more advanced topic/guide. Console example might be in some other repo. However, because the ASP.NET Core guide currently relies on prerelease packages, and that doesn't seem likely to change anytime soon, we feel like having the console guide close by will be useful for some teams. We may have to live with the two for now. But I did add a banner on the ASP.NET Core guide with a "note" about the prerelease status.

The only open question for me is do we link to both from the main README? Don't have a strong opinion either way.

Reply:
No strong opinion either (slightly prefer having only one link instead of two, to keep the doc smaller).

Reply:
Not a strong opinion, but its not just the application type that is changing between 2 tutorials - it also changes manual span creation <-> instrumentation library creation. I do not have a neat alternative suggestion either, lets proceed and keep iterating if fresh ideas come up later.

Reply:
Sounds good, merged.